tests/integration/test_settings.py

```diff
@@ -173,9 +173,9 @@ def test_remoteconfig_sampling_rate_default(test_agent_session, run_python_code_
     )
     assert status == 0, err
 
-    events = test_agent_session.get_events()
-    events_trace_sample_rate = _get_telemetry_config_items(events, "DD_TRACE_SAMPLE_RATE")
-    assert {"name": "DD_TRACE_SAMPLE_RATE", "value": 1.0, "origin": "default"} in events_trace_sample_rate
+    events = test_agent_session.get_events(subprocess=True)
+    events_trace_sample_rate = _get_telemetry_config_items(events, "trace_sample_rate")
```

Comment:
Why did the value change here?

Reply:
Good catch. I made a mistake fixing a merge conflict, I need to revert this.